contracts/views/EulerGeneralView.sol

```diff
@@ -74,6 +74,12 @@ contract EulerGeneralView is Constants {
         uint eTokenBalanceUnderlying;
         uint dTokenBalance;
         IRiskManager.LiquidityStatus liquidityStatus;
+
+        // Overrides, duplicate entries possible
+        address[] overrideLiabilities;
```

Comment:
I guess its a question of where to push those calculations, my take is to push it offchain just because some providers charge per gas on view calls. No strong preference tbh

Reply:
right, but on the other hand you'll have ppl asking how to process that as it's not straightforward. let's discuss